lib/twingly/url.rb

```diff
@@ -123,7 +127,7 @@ def normalized_path
     end
 
     def valid?
-      addressable_uri && public_suffix_domain && SCHEMES.include?(normalized_scheme)
+      !!(scheme =~ ACCEPTED_SCHEMES)
```

Comment:
> It looks funny, seems like we only require a scheme to be valid, but I know that's not the case (this is the same as before). But I just wanted to write something about it.

My first attempt was actually this method returning `true` and I took care of this in `.internal_parse`, but then I realized that calling `#new` could lead to weird stuff (at least how it's currently implemented) so I went with the least amount of changes.